tmail-backend/data/data-rabbitmq/src/main/java/com/linagora/tmail/DisconnectorRequestSerializer.java

```diff
@@ -0,0 +1,82 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.tmail;
+
+import java.nio.charset.StandardCharsets;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.james.DisconnectorNotifier.AllUsersRequest;
+import org.apache.james.DisconnectorNotifier.MultipleUserRequest;
+import org.apache.james.DisconnectorNotifier.Request;
+import org.apache.james.core.Username;
+
+import com.google.common.base.Splitter;
+
+public class DisconnectorRequestSerializer {
+
+    public static class DisconnectorRequestSerializeException extends RuntimeException {
+        public DisconnectorRequestSerializeException(String message) {
+            super(message);
+        }
+
+        public DisconnectorRequestSerializeException(String message, Throwable cause) {
+            super(message, cause);
+        }
+    }
+
+    public static final String ALL_USERS_REQUEST = "[]";
+    public static final String USER_DELIMITER = ",";
+
+    public static String serialize(Request request) {
+        return switch (request) {
+            case MultipleUserRequest multipleUserRequest -> multipleUserRequest.usernameList().stream()
+                .map(Username::asString)
+                .collect(Collectors.joining(USER_DELIMITER, "[", "]"));
```

Comment:
Please use a standard format for the encoding (JSON) with a standard library encoding it. So that we do not bother with edge cases like `a,b@domain.com`.